Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
```

Comment:
### [coderay](http://coderay.rubychan.de)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.1 → 1.1.2

[_~~change-log~~, [source-code](http://github.com/rubychan/coderay)_]





<details>
<summary>Commits</summary>

A change of **54** commits. See the full changes on [the compare page](https://github.com/rubychan/coderay/compare/v1.1.1...v1.1.2).

These are the first 10 commits:
- (f06e0e3) [Add C++11 keywords](https://github.com/rubychan/coderay/commit/f06e0e319e13e0e4f1f90a7bdf634bc1ca66b182)
- (2612789) [Fixed typo](https://github.com/rubychan/coderay/commit/26127899acf888a1888195c8b50c4603167e2808)
- (92a8e56) [Small changes to grouping of new C++11 keywords](https://github.com/rubychan/coderay/commit/92a8e56ccd8d3aee05db9c7a821b24382528c674)
- (cee1223) [fix tests for Ruby 1.8](https://github.com/rubychan/coderay/commit/cee12238d0ca5ea0d9d1ae9ef0e4dbf80fdfc0dc)
- (2a4c37a) [Merge branch 'master' of https://github.com/rubychan/coderay](https://github.com/rubychan/coderay/commit/2a4c37a2a536e6d7b67ebe2ffe4d4f6a8fd1b42a)
- (e94cf8e) [fix Windows line breaks in FOLDERS file](https://github.com/rubychan/coderay/commit/e94cf8ea0e342f41837c4685a45c4d2af488fe6e)
- (dd142d2) [Fixed merge](https://github.com/rubychan/coderay/commit/dd142d29c860cb0c39ec30b25db6a8313310a09e)
- (54c94b7) [Add env setting to allow C extensions which are not allowed …](https://github.com/rubychan/coderay/commit/54c94b78f18939ef8517dd2818fbb699c0fa1391)
- (2bce2ac) [Allow failures with jruby 18 and 19 mode, RedCloth has a kno…](https://github.com/rubychan/coderay/commit/2bce2ac91ad7994f9ead130387c8b24905853064)
- (cd3bffb) [Shouldn't have removed the jruby lines](https://github.com/rubychan/coderay/commit/cd3bffb1086420c02774ba905a403f8efe313f46)


</details>